Adiciona tradução para inglês de arquivo ORGANIZATION-DAY.md (#13)
pugpe/meetup-cookbook: files/english/ORGANIZATION-DAY.md

# Organization on the day

>Here are some instructions and tips to be followed by the PUG-PE organizing members and volunteers on the event's day.

## Instructions

> ### Some things to be aware be of before the event's day

Reviewer: Sobrou aqui
```suggestion
### Some things to be aware of before the event's day
```

> It is important to the organizing member and volunteers to check if there are some items that might be necessary, just like: table, power plugs, video projector. It is wanted that, in case those items are not available, the members should look for alternate solutions so the meetup can happen in the best possible way for everyone.

Reviewer: Acredito que o uso do "e.g." em língua escrita é mais comum e funciona melhor do que o "just like". O que tu acha?

```suggestion
It is important to the organizing member and volunteers to check if there are some items that might be necessary (e.g., table, power plugs, video projector). It is wanted that, in case those items are not available, the members should look for alternate solutions so the meetup can happen in the best possible way for everyone.
```

---

### Show up early

> The one who are responsible for the organization of the meet up need to set up the event site, so, show up early! This way you can solve any unexpected issue calmly, set up the chairs, the video projector, greet the speakers and the participants.

Reviewer: Sugiro algumas modificações que modificam levemente a estrutura das sentenças, diferenciando um pouco do jeito que é normal em português mas pra mim fazem mais sentido em inglês. O que tu acha?

```suggestion
The people in charge of the meetup's organization need to set up the event site. So... Show up early! This way you can solve any unexpected issues calmly, set up the chairs, the video projector, greet the speakers and the participants.
```

> Lots of times the meetups happen, for example, in universities classrooms, and, most likely, not all participants know the site and end up getting lost and not finding the room where the event will be held. So, indicate well where the event will happen!

Reviewer: Só uma dúvida que fiquei ao ver esse trecho aqui e depois ver o documento original: isso era pra ser um novo parágrafo? Assim, pra mim parece fazer mais sentido ser um novo parágrafo

Reviewer 2: A princípio não tava como um parágrafo novo, mas pode se tornar um :man_shrugging:

Reviewer: Huheuehuehuehu, saquei. Era só uma dúvida mesmo. Se fosse pra modificar seria no documento original. Valeu :)

> Lots of times the meetups happen, for example, in universities classrooms, and, most likely, not all participants know the site and end up getting lost and not finding the room where the event will be held. So, indicate well where the event will happen!

Reviewer: Algumas sugestões sobre estrutura de sentenças. Tentando manter as sentenças mais curtas e com menos vírgulas (algo mais comum em português e menos em inglês).

Quanto à última sentença, sugiro mudar completamente pois acho que apenas "indicate well" não traz significado suficiente pra o que tá querendo ser passado.

```suggestion
Lots of times the meetups happen, for example, in universities classrooms. Most likely, not all participants know the place and may end up getting lost and not finding the room where the event will be held. So... Make sure there are directions to the event's site!
```

Reviewer: O que tu achou disso?

---

### PUG-PE introduction

> The PUG-PE is an event that is always welcoming new participants and lots of them, besides participating on the meet up, do not know the Python local community and, sometimes, not even what a community is. That's why it is really important that there is a brief introduction, lasting between 5 and 10 minutes, introducing what is a comunitty and the Python local community and what are their goals.

Reviewer: Honestamente não sei como poderia ficar melhor, só coloquei essa sugestão por ser a primeira coisa que veio à cabeça. Mas os dois "and" me levam a querer construir essa sentença de um jeito diferente. O que tu acha?
```suggestion
The PUG-PE is an event that is always welcoming new participants and lots of them, besides participating on the meet up, do not know the Python local community and, sometimes, not even what a community is. That's why it is really important that there is a brief introduction, lasting between 5 and 10 minutes, introducing: what is a community; the Python local community; and what are their goals.
```

Reviewer 2: @giuliocc dei uma alterada, vê o que tu achas 😛

Reviewer: Muito melhor :)

> The PUG-PE is an event that is always welcoming new participants and lots of them, besides participating on the meet up, do not know the Python local community and, sometimes, not even what a community is. That's why it is really important that there is a brief introduction, lasting between 5 and 10 minutes, introducing what is and what are the goals of a community and the Python local community.

Reviewer: Fico em dúvida se o melhor seria "introduction" ou "presentation" apenas pelo fato de já ter um "introducing" depois.

```suggestion
PUG-PE is an event that is always welcoming new participants and lots of them, besides participating on the meetup, do not know the Python local community and, sometimes, not even what a community is. That's why it is really important that there is a brief presentation, lasting between 5 and 10 minutes, introducing what is and what are the goals of a community and the Python local community.
```

Reviewer: O que tu achou disso?

Reviewer: Up

Reviewer 2: Bote fé, foi uma boa observação, vou alterar

Reviewer: Beleza. Fiquei dando UP pq não sabia se tu tava vendo ou não :)

> The PUG-PE is an event that is always welcoming new participants and lots of them, besides participating on the meet up, do not know the Python local community and, sometimes, not even what a community is. That's why it is really important that there is a brief introduction, lasting between 5 and 10 minutes, introducing what is and what are the goals of a community and the Python local community.

Reviewer: Não entendi a última sentença: "what is and what are the goals of a community and the Python local community".

Tu poderia explicar?

"what is" o que? "Python local community"? O que acho que entendi foi que a conexão entre as partes da sentença ficou quebrada e por isso não tá claro.
